hyperdome_server/common.py

```diff
@@ -39,6 +39,8 @@ class Common(object):
     The Common object is shared amongst all parts of OnionShare.
     """
 
+    version = '0.1'
```

Comment:
Might want to all caps, as a const. Up to you tbh

Reply:
This is overwritten elsewhere in the code anyway, removing the line because common is first on the refactor list.

Fixed.